ta4j-core/src/main/java/org/ta4j/core/analysis/criteria/NumberOfPositionsCriterion.java

```diff
@@ -33,18 +35,60 @@
  */
 public class NumberOfPositionsCriterion extends AbstractAnalysisCriterion {
 
+    private final PositionFilter positionFilter;
```

Comment:
This needs a Getter

Reply:
@team172011 I will do that after the best name for `PositionFilter` is selected. One general question about this PR (because I am a _little_ unsure): do you find this PR is an improvement in compare to the current version?